### PR TITLE
Mk 1248 changes to contracts collection

### LINKED ISSUE #1248: Changes to the Docs Smart Contract Collection
Some edits for the Contract Library and for the names of contracts used in the docs live examples origination script

1. rename the tzip16 off-chain one and two contracts to include the off-chain part, as this is the only difference between them
2.  add the known contracts for Integration tests, unless they are duplicates of ones already listed, in which case indicate that in the listing.
        - knownContract = hangzhounet:  KT1J5Vbek6SAgUStzb3HEMdrgRnMaanxNkNB 
            - to be called "LambdaViewWithStoredCounterContract"            
        -  knownBigMapContract is:  hangzhounet: KT1Ds2LC3QqxVVqh95VdfxTmBsRZZBrh8Pgm
           -    Not a supported FA1.2 contract.  :  Entrypoint "transfer" has type (pair (pair (address %0) (address %1)) (nat %2)), but 
           -    should have type  (pair address address nat). 
           -    Not an FA2 contract
    
3. Add more commentary for tzip and big map contracts
4. change the name from "Contract Library" to "contract Collection"
6. for the Storage, also show the Taquito version and use tabs to show Taquito/Michelson
7. add the new Fibonacci contracts from https://github.com/ecadlabs/taquito/pull/1251

### LINKED ISSUE #1262: Add new FIB OnChainView contracts to Taquito Live Docs Contract Collection and to Live Code Contract Origination Script
In Taquito Live Docs on the Contract Collection Page, add the two FIB contracts that are used in the live code examples for demonstrating On Chain View support.

Also, add the origination of these contracts to ~/taquito/example/deploy-integration-tests-contracts.ts

Give each contract an appropriate name

### LINKED ISSUE #1272: Update the picture in Taquito docs for Common Errors Messages to Kukai 2
The picture of Kukai in Common error messages on the transaction_limits.md page of Taquito docs needs to be updated to Kukai 2.

### PR DESCRIPTION
**Pull request by michaelkernaghan**
Closes #1248 
Closes #1262 
Closes #1272

First in a series of additions to the contract collection

### CODE REVIEW

File: docs/contract-test-collection.md [hunk 1]
```diff
@@ -127,6 +129,21 @@ storage (pair
           (nat %total_supply));
 ```
 
+## LambdaViewWithStoredCounterContract
```
Comment (roxaneletourneau) [inline docs/contract-test-collection.md:132]: I don't think this contract satisfy the tzip-4 view definition. Lambdaview in the name might be inaccurate:

> We define the following synonym in a contract's parameter type declaration:
> 
> ```
> view a r = (a, contract r)
> ```
> 
> A `view` is an entrypoint which represents a computation that takes an argument of type `a` and returns a result of type `r`. This return type is represented as a callback.

Comment (michaelkernaghan) [reply docs/contract-test-collection.md]: I found the contract is the same as one originated in Live Code examples that is called "GenericMultisig"  so I have changed the name to "GenericMultisigContract"

Comment (michaelkernaghan) [reply docs/contract-test-collection.md]: I have changed the name to "GenericMultiSig" as the contract is the same as the one in Taquito Live Code Examples.

**Review by roxaneletourneau — APPROVED** [review]
